DESCRIPTION

```diff
@@ -17,5 +17,14 @@ Imports:
 Suggests:
     here,
     knitr,
-    testthat
+    testthat,
```

Comment:
I guess this is because of the current templates. Fine for now, but by release time we should probably simplify the templates (useful templates belong elsewhere) and thin out dependencies.

Reply:
this project has a loooooooong way to go before it gets released

Reply:
Also, this is related to the reason why #15 exists